frontend/src/static/css/style.scss

```diff
@@ -279,8 +280,8 @@ header{
 /* Authorship */
 #tab-authorship{
     .toolbar{
-        float: right;
-        cursor: pointer;
+        float:right;
+        cursor:pointer;
```

Comment:
avoid changing lines that are not part of your PR next time,
it complicate things in the event that PR needs to be reverted.

Reply:
noted